packages/applications/ssr/src/components/atoms/form/UploadDocument.tsx

```diff
@@ -50,7 +52,9 @@ const UploadNewDocument: FC<Omit<UploadDocumentProps, 'documentKey'>> = ({
     <div className={`fr-input-group ${className}`}>
       <label className="fr-label">
         {label}
-        <div className="fr-hint-text">Format accepté : {format}</div>
+        <div className="fr-hint-text">
+          Format accepté : {format}, taille maximale acceptée : {DEFAULT_FILE_SIZE_LIMIT_IN_MB} MB
```

Comment:
MB c'est de l'anglais, Mo en 🥖
```suggestion
          Format accepté : {format}, taille maximale acceptée : {DEFAULT_FILE_SIZE_LIMIT_IN_MB} Mo
```

Reply:
Ah oui exact